src/app/projects/services/doi.service.ts

```diff
@@ -0,0 +1,138 @@
+import {HttpClient} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+import {forkJoin, Observable, Subject, throwError} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {AutofillProject} from '../models/autofill-project';
+import {EuropePMCHttpSearchResponse, EuropePMCResult, Identifier} from '../models/europe-pmc-search';
+import {Project} from "@shared/models/project";
+import {IngestService} from "@shared/services/ingest.service";
+import {Router} from "@angular/router";
+import {AlertService} from "@shared/services/alert.service";
+
+@Injectable()
+export class DoiService {
+  API_URL = 'https://www.ebi.ac.uk/europepmc/webservices/rest/search';
```

Comment:
move to common config?

Reply:
I don't think that's needed. All environments query the EuropePMC webservice endpoint for getting project information using the doi